src/main/java/com/google/firebase/remoteconfig/internal/PublishOptions.java

```diff
@@ -0,0 +1,33 @@
+package com.google.firebase.remoteconfig.internal;
```

Comment:
This can probably live in the parent package as a package-protected class.

Reply:
Good call! Moved to the parent class and made package protected, including the ctor. Thanks!